types-of-contributions.md

```diff
@@ -0,0 +1,20 @@
+## What are the ways in which I can contribute to this project?
+
+This might be the question in your head when you were going through the code of AMIBOT.
+
+Well, its very simple.
+
+#### If you are a complete beginner, you can just open up the readme file or any such documentation file and look for any typing or grammatical errors in them. 
```

Comment:
Add a point about finding, reporting bugs, usability improvements or new features. Creating issues is a great way to contribute!